app/services/idv/flows/capture_doc_flow.rb

```diff
@@ -2,9 +2,7 @@ module Idv
   module Flows
     class CaptureDocFlow < Flow::BaseFlow
       STEPS = {
-        mobile_front_image: Idv::Steps::MobileFrontImageStep,
```

Comment:
We're gonna wanna keep these steps for now and use the feature flag to determine which ones the flow uses.

Reply:
Yeah, wasn't too clear on what of this to worry about. Also mentioned in the original comment:

> This does not yet remove any now-unused hybrid flow steps / views. Nor does it add support for analytics associated with previous hybrid flow (submissions, drop-offs, etc). Nor does it account for a scenario that `FeatureManagement.document_capture_step_enabled?` is `false`
> * Unclear what of these should be implemented here, vs. later, vs. never (in the case that `document_capture_step_enabled?` would assume to always be `true`)

Partly, I was wondering if we're at the point where we can start assuming a default switch-over to the new flow. But even if that were the case, I assume we'll want some control to switch back if needed in the short term.

As it impacts some of the other changes, I guess as long as we're always sending the session UUID, we don't necessarily need to check the feature flag when validating it in `process_result`, even if ultimately we're not really using it yet.

Reply:
Feature flag consideration added in rebased b5f1334ea68533771810bbf9b83022da279d3482.